pages/ksphere/kommander/latest/quick-start/index.md

```diff
@@ -31,7 +33,7 @@ Before starting the Konvoy installation, you should verify the following:
 
 ### Download and Install
 
-Download Konvoy with Kommander [here](https://github.com/mesosphere/konvoy/releases).
+Download [Konvoy with Kommander](https://github.com/mesosphere/konvoy/releases).
 
 Download the tarball to your local Downloads directory.
```

Comment:
Aren't these the same instructions from Konvoy at this point? If so, we should remove this whole section and point them to the Konvoy docs.